src/components/main-page/Partners/Partners.module.scss

```
@@ -0,0 +1,43 @@
+.heading {
+  margin-bottom: var(--title-spacing);
```

It should be:
```suggestion
  margin-bottom: var(--font-size-2xl);
```
? Or new variable with `48px` as in Figma mockup

---

That's for spotting it! Actually I was copy-pasting it from another section so I will fix it there too 😁